runtime/lua/vim/lsp/client.lua

```diff
@@ -671,13 +671,32 @@ function Client:_request(method, params, handler, bufnr)
   bufnr = resolve_bufnr(bufnr)
   log.debug(self._log_prefix, 'client.request', self.id, method, params, handler, bufnr)
   local success, request_id = self.rpc.request(method, params, function(err, result)
+    -- Per LSP, don't show ContentModified error to the user.
+    if err and err.code ~= lsp.protocol.ErrorCodes.ContentModified then
+      vim.notify(
+        string.format('[LSP][%s] %s: %s', self.name, err.code, err.message),
```

Comment:
(not a blocker) the [LSP] format is not our usual message format and seems extraneous. 

```suggestion
        string.format('LSP: %s (%s): %s', self.name, err.code, err.message),
```